pkg/minikube/command/kic_runner.go

```diff
@@ -185,7 +186,11 @@ func (k *kicRunner) Copy(f assets.CopyableFile) error {
 		}
 	}
 	klog.Infof("%s (temp): %s --> %s (%d bytes)", k.ociBin, src, dst, f.GetLength())
-	tf, err := ioutil.TempFile("", "tmpf-memory-asset")
+	tmpFolder := ""
+	if isSnapBinary() {
```

Comment:
add comment with the link to the issue, that why we are doing this.

Reply:
Done